recipes/google-cloud-cpp-compute/bld.bat

```diff
@@ -0,0 +1,25 @@
+@echo on
+setlocal EnableDelayedExpansion
```

Comment:
I see no variables defined in if-branches, so I don't think we actually need this line?

Reply:
Removed.